docs/eventing/sink/kafka-sink.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Apache Kafka Sink"
```

Comment:
It's not really clear in this section whether it's supposed to explain concepts or teach a user to do individual tasks like install, etc. There seem to be a few procedures jumbled in. Can we make that clearer?